learn/cloud/monitoring.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Monitoring metrics
+description: Meilisearch Cloud offers in-depth search analytics to help monitor your application performance.
+sidebarDepth: 3
+---
+
+# Monitoring
+
+Enable Meilisearch Cloud monitoring to keep track of application performance and service status. 
+
+## Requirements
+
+You must have a [Meilisearch Cloud](https://meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=monitoring) account to access monitoring metrics.
+
+## Activating monitoring
+
+### Enable monitoring in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics and monitoring" section and click on the "Enable analytics monitoring" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics and monitoring", and a short explanation of both features.](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/cloud-analytics/01-analytics-enable.png)
```

Comment:
I assume the image will be updated after the feature release as it serves both guides?

Reply:
yes, the images will be updated as soon as I can grab screenshots of this section of the project overview.

Reply:
i don't know if the screenshot should be good by now, so i'm mentioning this again in case it was overseen